tests/integration/components/menu-test.js

```diff
@@ -64,6 +64,20 @@ function assertNoActiveMenuItem(menuSelector) {
   QUnit.assert.dom(menuSelector).doesNotHaveAria('activedescendant');
 }
 
+async function type(selector, value) {
+  let finalKeyEventProcessing;
+
+  for (const char of value) {
+    finalKeyEventProcessing = triggerKeyEvent(
+      selector,
+      'keydown',
+      char.toUpperCase()
+    );
+  }
+
+  await finalKeyEventProcessing;
+}
+
```

Comment:
Just wanted to throw an explanation in here, since this is a little weird:

We need a custom way to trigger a sequence of key events because Ember's `fillIn` and `typeIn` helpers reject sending input to an element that isn't an `input` or `contenteditable`.

We only want to await the _final_ key event, otherwise we will wait the 350ms after _each_ event for the search term to be cleared, which is _not_ what we want.

I tried to implement the Ember Concurrency task using `rawTimeout` (which the tests will _not_ wait for) but then ran into a problem where it's quite for the test to wait for the search term to clear when we _do_ want it to, since in some tests we search for multiple items in a row. 

TL;DR: I tried to clean this up but the other approaches I could think of didn't actually work in practice!